### PR TITLE
Headers Swedish old translations fixed

### PR DESCRIPTION
**Pull request by adoroshk**
### Summary
This PR replaces **terra-notification-dialog** headers Swedish translations for locales **sv** and **sv-SE** to match the new translations used elsewhere.

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [x] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### New headers for for locales **sv** and **sv-SE**  locales.
Screenshot 2024-04-15 at 9 43 41 AM
Screenshot 2024-04-15 at 9 43 50 AM
Screenshot 2024-04-15 at 9 44 00 AM
Screenshot 2024-04-15 at 9 44 08 AM

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**
UXPLATFORM-10362

### CODE REVIEW

File: packages/terra-notification-dialog/translations/sv-SE.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "Terra.notification.dialog.hazard-low": "Information",
-  "Terra.notification.dialog.error": "Fel",
-  "Terra.notification.dialog.hazard-medium": "Varning",
-  "Terra.notification.dialog.hazard-high": "Avisering"
+  "Terra.notification.dialog.hazard-low":	"Information",
+  "Terra.notification.dialog.error":	"Fel",
+  "Terra.notification.dialog.hazard-medium":	"Obs!",
+  "Terra.notification.dialog.hazard-high":	"Varning"
```
Comment (cm9361) [inline packages/terra-notification-dialog/translations/sv-SE.json:5]: It seems like extra white space was added to the strings.

File: packages/terra-notification-dialog/translations/sv-SE.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "Terra.notification.dialog.hazard-low": "Information",
-  "Terra.notification.dialog.error": "Fel",
-  "Terra.notification.dialog.hazard-medium": "Varning",
-  "Terra.notification.dialog.hazard-high": "Avisering"
+  "Terra.notification.dialog.hazard-low":	"Information",
```
Comment (cm9361) [inline packages/terra-notification-dialog/translations/sv-SE.json:2]: Is this the proper translated word?  It appears to be English.

File: packages/terra-notification-dialog/translations/sv-SE.json (hunk 1 above)
Comment (adoroshk) [reply packages/terra-notification-dialog/translations/sv-SE.json]: Good catch, removed extra spacing in https://github.com/cerner/terra-framework/pull/2136/commits/3b3f5cb89b3b6b8fc984399a530c3ac5bc9bb7d7

File: packages/terra-notification-dialog/translations/sv-SE.json (hunk 2 above)
Comment (adoroshk) [reply packages/terra-notification-dialog/translations/sv-SE.json]: It was provided that way:
Screenshot 2024-04-15 at 10 22 58 AM

And it looks like it's the same in English and Swedish:
Screenshot 2024-04-15 at 10 25 11 AM

File: packages/terra-notification-dialog/translations/sv.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-  "Terra.notification.dialog.hazard-low": "Information",
+  "Terra.notification.dialog.hazard-low":	"Information",
```
Comment (sycombs) [inline packages/terra-notification-dialog/translations/sv.json:2]: Should that just be a space?
```suggestion
  "Terra.notification.dialog.hazard-low": "Information",
```

**Review by sycombs — APPROVED** [review]

File: packages/terra-notification-dialog/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+ * Headers Swedish old translations replaced to match the new translations used elsewhere.
```
Comment (sdadn) [inline packages/terra-notification-dialog/CHANGELOG.md:6]:
```suggestion
 * Fixed Swedish translations for headers with new translations used elsewhere.
```

**Review by sdadn — APPROVED** [review]

**Pull request merged**